pkg/agent/route/route_linux.go

```diff
@@ -198,6 +208,12 @@ func (c *Client) initIPTables() error {
 			"-i", c.hostGateway, "-d", c.serviceCIDR.String(),
 			"-j", iptables.MarkTarget, "--set-xmark", rtTblSelectorMark,
 		}...)
+		writeLine(iptablesData, []string{
+			"-A", antreaMangleChain,
+			"-m", "comment", "--comment", `"Antrea: unmark post LB service packets"`,
+			"-i", c.hostGateway, "!", "-d", c.serviceCIDR.String(),
+			"-j", iptables.MarkTarget, "--set-xmark", "0/0xffffffff",
```

Comment:
should it only reset the bit used for service mark? otherwise other marks by kubernetes will be reset too.

Reply:
I think we want to clean *all* marks for re-entrance traffic, because we do not want to have marks from previous entrance (i.e LB) interfere with current (forwarding) entrance.
Also it is set in PREROUTING stage mangle table, any subsequent K8s marking should not be impacted by this.

Reply:
Got it.